src/components/ProfileValidation/ReturnToAppButton.tsx

```diff
@@ -0,0 +1,27 @@
+import { Button } from '@mui/material'
```

Comment:
if no children it renders a 'cancel' button to return to account or redirectURL if it exists
if there is a child element it wraps it in to do the same onClick